### DISCUSSION TITLE
Adding a 12 step program to the 12 step programs list

### DISCUSSION THREAD

**Discussion by Underearners-Anonymous** [code4recovery/12-step-meeting-list#1164] [Q&A] [answered]
How can we get the program Underearners Anonymous listed as an option on the 12-step meeting list plugin? Thanks

**Comment by anchovie91471** [comment]
@Underearners-Anonymous Thanks for reaching out! 

We will open an issue to get Underearners Anonymous added to the list. Can you provide a list of the types of meetings you want available when UA is chosen? You can find a sample list at https://github.com/code4recovery/spec to get you started.

Once we get this list, I'll open the issue to get it added.

**Comment by Underearners-Anonymous** [comment]
Great! Thank you. I'll have a look and get back to you with that list.


Clara T

President, UA GSB, Inc.

Chair of the UA General Service Board

UA Website Manager

**Comment by Underearners-Anonymous** [comment]
Dear Anthony,

Thank you for your help. Here is a list of types of meetings there are in
Underearners Anonymous. It is not an exhaustive list. Will our developer be
able to add or delete any of these options or would we have to contact you?
Thanks

In-person

Phone

Video

Steps

Tools

Women’s

Men’s

BIPOC

Spanish speaking

Thanks


Clara T

President, UA GSB, Inc.

Chair of the UA General Service Board

UA Website Manager

**Comment by joshreisner** [comment, marked as answer]
hi @Underearners-Anonymous 

your program has been added in yesterday's release `3.14.17`

our plugin handles in-person and online based on data entered, not by adding specific types to the meetings manually -- we implemented it this way to reduce confusion when the information says one thing but the types say another. 

we are happy to add more types for you in the future, or you may add them to your site directly by [following the instructions here](https://wordpress.org/plugins/12-step-meeting-list/#my%20meeting%20type%20isn%27t%20listed%21)

**Comment by Underearners-Anonymous** [comment]
Great! Thank you Josh!

In service and appreciation,
Clara T, UAGSB, Inc. President/GSB Chair
UA Website Manager
